FLIR/conservator/generated/schema.py

```diff
@@ -19,12 +19,12 @@ class AllowedLabelCharacters(sgqlc.types.Scalar):
 
 class AnnotationImportState(sgqlc.types.Enum):
     __schema__ = schema
-    __choices__ = ("completed", "failed", "processing", "retrying", "uploading")
+    __choices__ = ("uploading", "processing", "retrying", "completed", "failed")
 
```

Comment:
It looks like you have an old version of sgqlc in your venv, because there is a lot of churn in the schema from un-sorting choices again. The current version is 13.0 and I think I will add a min version to pip requirements since it did change the ordering from earlier versions.